### ISSUE TITLE
Ambiguous information in how to build adaptive card tabs

### ISSUE THREAD

**Issue by florianwachter** [MicrosoftDocs/msteams-docs#8585]
The information in the how to [how-to](https://github.com/MicrosoftDocs/msteams-docs/tree/main/msteams-platform/tabs/how-to)/build-adaptive-card-tabs.md is ambiguous: The warning says: "Adaptive Card tabs will be deprecated in the new Microsoft Teams. Apps are expected to be available in the new Microsoft Teams by June 2023. If your app is using Adaptive Card tabs, it's recommended to rebuild the tab as a web-based tab. For more information, see Build tabs for Teams."

And a few lines below it means: "Adaptive Card tabs are a new way to build tabs in Teams."

What is it now? Is it recommended or will it be deprecated?

Here is the link to the article: [https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs)

**Comment by Meghana-MSFT** [comment]
@florianwachter - Apologies for the ambiguity. Please note that "Adaptive Card tabs will be deprecated in the new Microsoft Teams."

The same will be mentioned in the samples as well - https://github.com/OfficeDev/Microsoft-Teams-Samples/issues/794

"Adaptive Card tabs are a new way to build tabs in Teams." was added as a definition for Adaptive card tabs.

**Comment by florianwachter** [comment]
Hi Meghana,
Thanks for your reply. So adaptive card tabs will no longer be supported in MS teams? I just want to make sure, since we already invested a lot of time developing adaptive card tabs, as they were promoted the new way to build static tabs in MS teams. Can you please confirm that web-based tab showing in an IFRAME will be the only valid way in the future?

Thanks, Florian

**Comment by Meghana-MSFT** [comment]
Yes, Adaptive Card tabs will be deprecated in the new Microsoft Teams. 

> Can you please confirm that web-based tab showing in an IFRAME will be the only valid way in the future?

Yes.

**Comment by florianwachter** [comment]
Thanks!
